Kickstarter-iOS/Views/Controllers/RewardPledgeViewController.swift

```diff
@@ -305,6 +305,7 @@ internal final class RewardPledgeViewController: UIViewController {
     _ = self.orLabel
       |> UILabel.lens.font .~ .ksr_footnote()
       |> UILabel.lens.textColor .~ .ksr_dark_grey_500
+      |> UILabel.lens.text %~ { _ in "– or –" }
```

Comment:
Not worth adding a `localizedString(...)` here in the mean time? I usually search the project for occurrences of `localizedString` from time to time to check that we are using the `Strings` functions.